tests/test_mapping/test_mapping_helpers.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+# Local Imports
+from src.mapping.mapping_helpers import update_ref_list
+
+
+class TestUpdateRefList(object):
+    """Tests for update_ref_list."""
+
+    @pytest.fixture(scope="function")
+    def full_input_df(self):
+        """Main input data for update_ref_list tests."""
+        columns = ["reference", "instance", "formtype", "cellnumber"]
```

Comment:
I think we should have selectiontype in here as well, so we don't have nulls in the expected dataframe at the end//